src/rtScriptDuk/debugger/duk_debugger_interface_windows.c

```diff
@@ -0,0 +1,376 @@
+#undef UNICODE
```

Comment:
Let's rename to duk_debugger_interface.c and add it to rtScriptDuk/debugger/win/.